app/components/avo/fields/belongs_to_field/edit_component.rb

```diff
@@ -57,6 +57,17 @@ def field_html_action
     @field.get_html(:data, view: view, element: :input).fetch(:action, nil)
   end
 
+  def create_path(target_resource = nil)
+    return nil if @resource.blank?
+
+    helpers.new_resource_path(**{
+      via_relation: @field.id.to_s,
+      resource: target_resource || @field.target_resource,
+      via_resource_id: resource.model.id,
```

Comment:
Should we use `resource.model.to_param` instead?

Reply:
Good catch! Looks like that's also happening here: 

https://github.com/avo-hq/avo/blob/c94dc00eac0f8bbd4a79055868c96521305aaac2/app/components/avo/fields/belongs_to_field/show_component.rb#L8-L9